pkg/process/config/yaml_config.go

```diff
@@ -94,6 +94,19 @@ func (a *AgentConfig) LoadProcessYamlConfig(path string) error {
 	a.setCheckInterval(ns, "process_realtime", RTProcessCheckName)
 	a.setCheckInterval(ns, "connections", ConnectionsCheckName)
 
+	if a.CheckIntervals[ProcessCheckName]%a.CheckIntervals[RTProcessCheckName] != 0 {
```

Comment:
Should there be code to also make sure regular check interval is always greater than RT check?

Reply:
Great point I will add that.